coriolis/osmorphing/debian.py

```diff
@@ -21,13 +34,89 @@ def _check_os(self):
                 debian_version_path).decode().split('\n')[0]
             return ('Debian', release)
 
+    def disable_predictable_nic_names(self):
+        grub_cfg = os.path.join(
+            self._os_root_dir,
+            "etc/default/grub")
+        if self._test_path(grub_cfg) is False:
+            return
+        contents = self._read_file(grub_cfg).decode()
+        cfg = utils.Grub2ConfigEditor(contents)
+        cfg.append_to_option(
+            "GRUB_CMDLINE_LINUX_DEFAULT",
+            {"opt_type": "key_val", "opt_key": "net.ifnames", "opt_val": 0})
+        cfg.append_to_option(
+            "GRUB_CMDLINE_LINUX_DEFAULT",
+            {"opt_type": "key_val", "opt_key": "biosdevname", "opt_val": 0})
+        cfg.append_to_option(
+            "GRUB_CMDLINE_LINUX",
+            {"opt_type": "key_val", "opt_key": "net.ifnames", "opt_val": 0})
+        cfg.append_to_option(
+            "GRUB_CMDLINE_LINUX",
+            {"opt_type": "key_val", "opt_key": "biosdevname", "opt_val": 0})
+        self._write_file_sudo("etc/default/grub", cfg.dump())
+        self._exec_cmd_chroot("/usr/sbin/update-grub")
```

Comment:
Apart from this last `update-grub` call I believe all of the above code also applies for RHEL-based distros so we could abstract it (or the `update-grub` call) away in a method in the base/child classes.
LE: nvm I wasn't aware of the `new-kernel-pkg` binary on RHELs